src/java/test/org/apache/zookeeper/server/quorum/CommitProcessorConcurrencyTest.java

```diff
@@ -106,6 +106,7 @@ public void shutdown() {
 
                 @Override
                 public void notifyStopping(String threadName, int errorCode) {
+                    Assert.fail("Commit processor crashed " + errorCode);
```

Comment:
I wonder if you have any insight on what does his test do and mine don't? I thought it was pretty much the same